js/src/features/human-resouce/components/HumanResourceCalendar/HumanResourceCalendar.tsx

```diff
@@ -1,72 +1,117 @@
 
 type Props = {
   employees: EmployeeShort[];
   absences: AbsencesMap;
+  vacations: VacationMap;
 };
 
 export const HumanResourceCalendar: React.FC<Props> = ({
   employees,
   absences,
+  vacations,
 }) => {
   const tableRef = useRef<HTMLTableElement>(null);
 
-  const [{ date, daysCount }, dispatch] = useReducer(reducer, initial);
+  const [{ date, daysCount, scrollDaysCount }, dispatch] = useReducer(
+    reducer,
+    initial,
+  );
   const { interval, format, title } = getDateInterval(date, daysCount);
   const months = getMonthsFromInterval(interval);
 
   return (
     <div className="content human-resource-calendar">
-      <LeftOutlined
-        style={{ userSelect: "none" }}
-        onClick={() => dispatch({ type: "move", direction: "back" })}
-      />
-      <RightOutlined
-        style={{ userSelect: "none" }}
-        onClick={() => dispatch({ type: "move", direction: "forward" })}
-      />
-      <table className="timetable" ref={tableRef}>
+      <table
+        className="timetable"
+        ref={tableRef}
+        onWheel={(e) => {
+          dispatch({
+            type: "scroll",
+            daysCount: Math.sign(e.deltaY) * scrollDaysCount,
+          });
+        }}
+      >
         <thead>
           <tr>
             <th className="timetable-title" rowSpan={2}>
               {title}
             </th>
+            <th />
             {months.map(([month, days]) => (
-              <th key={month} colSpan={days.length}>
+              <th
+                key={month}
+                colSpan={days.length}
+                className="timetable-title__month"
+              >
                 {month}
               </th>
             ))}
           </tr>
           <tr>
+            <th>
+              <LeftOutlined
+                style={{ userSelect: "none" }}
+                onClick={() => dispatch({ type: "move", direction: "back" })}
+              />
+            </th>
             {interval.map((date) => (
               <th key={date.getTime()} className="timetable-date">
                 {formatDate(format, date)}
               </th>
             ))}
+            <th>
+              <RightOutlined
+                style={{ userSelect: "none" }}
+                onClick={() => dispatch({ type: "move", direction: "forward" })}
+              />
+            </th>
+            <th>Vacation</th>
+            <th>Sick</th>
           </tr>
         </thead>
         <tbody>
           {employees.map((employee) => (
-            <tr key={employee.userId}>
-              <td className="timetable-user">
-                <HumanResourceCalendarUser employee={employee} />
-              </td>
+            <tr key={employee.userId} className="select-row">
+              <th className="timetable-user">
+                <HumanResourceCalendarUserAvatar employee={employee}>
+                  <HumanResourceCalendarUser employee={employee} />
```

Comment:
Why is it a child of `HumanResourceCalendarUserAvatar`. Maybe we should unite this component with `HumanResourceCalendarUserAvatar`?

Reply:
I think it's good to use composition, as there is a possibility to reuse `HumanResourceCalendarUserAvatar`

Reply:
If you extract `initials` as function in helper your `HumanResourceCalendarUserAvatar` will consist only of `antd.Popover`. And the same with `HumanResourceCalendarUser`. It is just a link. We can argue but I think there is not enough functionality for a component. But if you unite this two components - it'll be a good meaningful component.
If in the future if there will be a need to reuse you can separate it in any moment. Don't overcomplicate.

Reply:
Done